src/cs/lib/msquic.cs

```diff
@@ -10,12 +12,22 @@
 
 namespace Microsoft.Quic
 {
-    public unsafe partial struct QUIC_BUFFER
+#if PUBLIC_MSQUIC_INTEROP_ACCESS
```

Comment:
Is this still necessary when you have `InternalsVisibleTo` ? Or is there a reason for different approach in this file and `*_generate*.cs`?

Reply:
How does `InternalsVisibleTo` work? That sounds like it might be a better solution.

Reply:
It makes types/fields/methods marked `internal` to look like `public` for assemblies named in the `InternalsVisibleTo` attribute. Disadvantage is that your interop lib must know the consumer lib/executable name beforehand.

Reply:
Yeah I meant to revert these back. For our test specifically InternalsVisableTo is fine.